staging/knative/charts/serving/templates/serving-core.yaml

```diff
@@ -2539,7 +2538,7 @@ spec:
         - name: webhook
           # This is the Go import path for the binary that is containerized
           # and substituted here.
-          image: gcr.io/knative-releases/knative.dev/serving/cmd/webhook@sha256:bc13765ba4895c0fa318a065392d05d0adc0e20415c739e0aacb3f56140bf9ae
+          image: gcr.io/knative-releases/knative.dev/serving/cmd/webhook:{{ .Chart.AppVersion }}
```

Comment:
i tested this quickly and it looks like the AppVersion tag (`v0.22.3`) isn't the same image as specified by the SHA

Reply:
nvm i used the wrong AppVersion